engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-system-purpose-modal.html

```diff
@@ -0,0 +1,80 @@
+<div data-extend-template="components/views/bst-modal.html">
+  <h4 data-block="modal-header" translate>Content Host System Purpose</h4>
+
+  <div data-block="modal-body">
+    <h4 translate>Assign System Purpose:</h4>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div bst-global-notification></div>
+      </div>
+    </div>
+
+    <div bst-alert="info" ng-show="showConfirm">
+      <span translate>
+        Are you sure you want to set the System Purpose values on the content host(s) selected?
+      </span>
+      <div>
+        <button type="button" class="btn btn-default" ng-click="showConfirm = false; performAction()" translate>Yes</button>
+        <button type="button" class="btn btn-default" ng-click="showConfirm = false; cancel()" translate>No</button>
+      </div>
+    </div>
```

Comment:
Move the confirmation blob to the bottom of the modal. When the user clicks the first Assign button, it is replaced by the confirmation blob. 

- Updated and it does this now

The confirmation blob also covers / replaces the modal's Cancel button - Since moving it to the bottom I am not hitting this. 

- Tried with Opera and Firefox

In the confirmation blob, change the button titles to 'Assign' and 'Cancel'. The 'Assign' button should have class btn-primary and 'Cancel' should have class btn-default (in other words, reverse the default and primary). The different colors will make it more clear that 'Assign' and 'Cancel' are very different actions.

- Removed the `no` from the confirmation blob and just left 'assign' with the new button color/class. I kept cancel at the bottom right and changed the class to `default` Working on the other feed back.

Reply:
> The confirmation blob also covers / replaces the modal's Cancel button

sorry, I meant that the confirmation blob _should_ cover or replace the modal's Cancel button, so that you don't have two Cancel buttons on screen.  Looks like you've addressed this now, thx :slightly_smiling_face: